communicator/ssh/communicator.go

```diff
@@ -812,3 +812,8 @@ func (c *bastionConn) Close() error {
 	c.Conn.Close()
 	return c.Bastion.Close()
 }
+
+func quoteFilesystemPath(path string) string {
+	replacedQuotes := strings.ReplaceAll(path, "'", "")
```

Comment:
I think instead of removing quotes, we can just do proper escaping.
Looks like there are some go libs that are doing it e.g. https://github.com/alessio/shellescape
And some resources, how to do it properly:
https://stackoverflow.com/questions/34898430/escaping-for-proper-shell-injection-prevention
https://qntm.org/bash

Reply:
I added an implementation of `shlex.quote`